macros/models/_sector/erc4337/erc4337_userops_basics.sql

```diff
@@ -10,6 +10,7 @@ with userop as (
     select
           '{{ blockchain }}' as blockchain
         , '{{ version }}' as version
+        , cast(date_trunc('month', evt_block_time) as date) as block_month
```

Comment:
need to ensure we cast partition column explicitly, as `date_trunc` doesn't appear to do so on trino
partitions must be date, if a date/time field, not timestamp